client/src/pages/SearchResultsUser.js

```diff
@@ -11,6 +12,24 @@ function SearchResultsUser() {
   }
 
   const query = useQuery().get("query");
+  componentDidMount() {
```

Comment:
I would wrap this in the useEffect method that we've been using in class, if you look at my PR you can see how I wrapped mine in it. 
ComponentDidMount is one of those older methods for when we didn't do functional-based components and did class based.
When you switch to useEffect, make sure you pass it a [ ] as the second  parameter like in mine as that makes sure it only goes off once (according to Sarah)